strawberry/printer.py

```diff
@@ -1,7 +1,215 @@
-from graphql.utilities.schema_printer import print_type as graphql_print
+from itertools import chain
```

Comment:
this has been copy and pasted from graphql core, we can remove the duplicated functions, but we did change some of them :)

Reply:
From what I remember GraphQL-core tries to stick to the same API as GraphQL.js, so I don’t think they will add this feature.

Also I think it might be worth writing our own printer, as maybe in future we might migrate away from GraphQL-core!